templates/rpm.erb

```diff
@@ -184,6 +184,10 @@ fi
 %preun
 if [ "${1}" -eq 0 ]
 then
+<%# Making sure that at least one command is in the function -%>
+<%# avoids a lot of potential errors, including the case that -%>
```

Comment:
I'm OK with this change, but I wonder if we should just move this logic into the script() method? Something to ponder.